benefits/templates/registration/logged_out.html

```diff
@@ -0,0 +1,35 @@
+{% extends "admin/agency-base.html" %}
+{% load static %}
+
+{% block title %}
+    Logged out | Cal-ITP Benefits Administrator
+{% endblock title %}
+
+{% block extrastyle %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static "admin/css/login.css" %}">
+{% endblock extrastyle %}
+
+{% block bodyclass %}
+    {{ block.super }} login
+{% endblock bodyclass %}
+
+{% block branding %}
+    <div class="d-flex justify-content-center w-100 bg-primary">
+        <img class="my-3" src="{% static "img/logo-lg.svg" %}" width="220" height="50" alt="California Integrated Travel Project: Benefits logo (large)" />
+    </div>
+
+    <div id="site-name">
+        <h1 class="text-center text-white fs-3 py-3 m-0">Administrator</h1>
+    </div>
```

Comment:
This is the same exact code as https://github.com/cal-itp/benefits/blob/main/benefits/templates/admin/login.html#L21-L29 so I _could_ turn it into an `includes`... but I didn't think we'd re-use this a third time, so I opted not to.

Reply:
Yep, I can't see where we would re-use it since it's specific to the log in and log out pages.